### PR TITLE
Adding event grid support

### PR DESCRIPTION
**Pull request by Jumaodhiss**
We are adding Azure event Grid as a change notification delivery option

### CODE REVIEW

**Comment by Lauragra** [comment]
@FaithOmbongi , can you review and approve this one please? Thanks.

File: concepts/change-notifications-delivery.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Steps:
 
 After you create the required Azure KeyVault and Azure Event Hubs services, you will be able to create your subscription and start receiving change notifications via Azure Event Hubs.
 
-#### Creating the subcription
+#### Creating the subscription
```
Comment (FaithOmbongi) [inline concepts/change-notifications-delivery.md:123]:
```suggestion
#### Create the subscription
```

File: concepts/change-notifications-delivery.md [hunk 2]
```diff
@@ -184,11 +184,79 @@ Connect-Graph -Scopes "Application.ReadWrite.All"
...
+### Getting started with Azure Event hubs
 
 See the following Azure Event Hubs quick starts:
 
 - [.NET Core](/azure/event-hubs/get-started-dotnet-standard-send-v2)
 - [Java](/azure/event-hubs/event-hubs-java-get-started-send)
 - [Python](/azure/event-hubs/get-started-python-send-v2)
 - [JavaScript](/azure/event-hubs/get-started-node-send-v2)
+
+## Using Azure Event Grid to receive change notifications (Preview)
```
Comment (FaithOmbongi) [inline concepts/change-notifications-delivery.md:196]: I suggest you move everything from this line to a separate article dedicated to Event Hub/Grid. Similar to what's live in the TOC for cloud printing, Teams, and Outlook.
```suggestion
## Use Azure Event Grid to receive change notifications (Preview)
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @Jumaodhiss,

Haven't yet reviewed the PR but a few suggestions.
- Move all the info from line 196 into a separate conceptual article. You can end the article with a "Next steps" with info from line 187-194.
- Please resolve all warnings and suggestions - All links should be relative links. For example, instead of `https://docs.microsoft.com/en-us/azure/event-grid/compare-messaging-services#azure-event-grid`, use `/azure/event-grid/compare-messaging-services#azure-event-grid`.

**Pull request closed without merging**